src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--1 > :nth-child(-n+1) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--2 > :nth-child(-n+2) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--3 > :nth-child(-n+3) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--4 > :nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--5 > :nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
```

Comment:
Group the selectors with the same properties.
Or, even better, use `for` rule from Sass
https://sass-lang.com/documentation/at-rules/control/for/

Reply:
It looks great, thanks for advice!